packages/strapi-parts/src/Table/Table.js

```diff
@@ -70,8 +73,8 @@ export const Table = ({ colCount, rowCount, footer, ...props }) => {
   }, []);
 
   return (
-    <Box shadow="tableShadow">
-      <TableBox background="neutral0" overflowing={overflowing}>
+    <Box shadow="tableShadow" hasRadius>
+      <TableBox footer={footer} background="neutral0" overflowing={overflowing}>
```

Comment:
Can you rename the TableBox props please:

Something like 
```js
fullBorderRadius={footer}
or
withBorderRadius={footer}
or 
hasFullBorderRadius={footer}
🤷🏽‍♂️
```

Reply:
I'd prefer something that starts by `with` ....

Reply:
let's go for with